src/test/java/jalau/at18/architects/TestGrayCard.java

```diff
@@ -1,7 +1,10 @@
 package jalau.at18.architects;
 
 import org.junit.Test;
-import org.junit.Assert;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
 public class TestGrayCard {
```

Comment:
In order to keep consistency with other test files, please rename this class to `GrayCardTest`

Reply:
Hello @Solrac1906 , any updates on this one?

Reply:
fixed the observations